src/protocol-reference.rst

```diff
@@ -713,6 +713,23 @@ Streaming instructions
     :arg string filename:
         The name of the file, as it would be saved on a filesystem.
 
+.. guac:instruction:: msg
+    :sent-by: server
+    :phase: interactive
+
+    Sends a message from the server (gaucd) to the client. The nature of these
+    messages is intentionally ambiguous and flexible - the message must include
```

Comment:
I don't think "ambiguous" is quite right - each `msg` unambiguously represents a specific server-side event. Perhaps "broad"?

Reply:
Fixed via rebase.